src/functional/fencil_processors/builders/cpp/build.py

```diff
@@ -0,0 +1,207 @@
+# GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2014-2022, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+import importlib
+import pathlib
+import subprocess
+import textwrap
+from typing import Dict, Final, Optional, Sequence
+
+from eve import Node
+from eve.codegen import JinjaTemplate, TemplatedGenerator
```

Comment:
It's not super meaningful, but for historical reasons we import the templates like: `as_fmt`, `as_mako`, `as_jinja`. I suggest to preserve the convention for now.
```suggestion
from eve.codegen import JinjaTemplate as_jinja, TemplatedGenerator
```